### DISCUSSION TITLE
Can we use plane freely ?

### DISCUSSION THREAD

**Discussion by AlwaysNoobCoder** [makeplane/plane#4532] [Q&A] [answered]
We are a bank, we want to use plane internally.

Can we use it for free ?

**Comment by theparthacus** [comment]
Plane is both [self-hosted](https://docs.plane.so/self-hosting/overview) and on the [Cloud](app.plane.so/sign-up), so yes, for some features, you can use it Free. when you need more features or if you already anticipate needing those features, we recommend you buy a [Plane One](https://plane.so/one) license, our first self-hosted-only plan. more features will come to Pro and Ultimate for both self-hosted instances and the Cloud.

**Reply by Dhruv-Garg79** [reply]
@theparthacus is there a feature list comparison somewhere?

**Reply by TyronMYoung11** [reply]
sorry i got Confusion.
while this repo gains community help by `open-source` title, why self-hosted users should not be able to use all features as free?
it's logical that users must pay for cloud-hosted, as it has hosting costs, but this is wired about self-hosting the open-source project

**Reply by danciaclara** [reply]
We have two self-hosted editions, Community and Commercial. Please see [this](https://developers.plane.so/self-hosting/editions-and-versions) to understand the difference between the two.

**Reply by TyronMYoung11** [reply]
@danciaclara
Thanks, now i saw that.

Tell where to download the Community version? 
So it doesn't have that limitation on Seats?
So the integration is not locked in it?

**Reply by TyronMYoung11** [reply]
@danciaclara Is the page https://plane.so/pricing refers to Commercial only?
Can you append Community version to the pricing page?

**Comment by theparthacus** [comment]
we haven't been able to get around to that, but we soon will. meanwhile, [One](https://plane.so/one) should show you a decent comparison for what it unlocks over Free or Community.

**Reply by TyronMYoung11** [reply]
https://plane.so/one redirects to https://plane.so/pro

**Reply by theparthacus** [reply, marked as answer]
One is now gone. the plan closest to it is Pro which is what the page redirects to now.

**Comment by adamhub** [comment]
It's very ambiguous which features/limitations come with self hosted vs One. Is self-hosted limited to 50 users? Is everyone on the One page just available on One? And so on.  This issue is raised here also: https://www.reddit.com/r/selfhosted/comments/1fewz6v/feature_list_of_plane_selfhosted_vs_cloud/

**Reply by danciaclara** [reply]
Here's the [pricing page](https://plane.so/pricing?mode=self-hosted) that lists all the features in both the cloud and self-hosted plans.
Also, see [Editions](https://developers.plane.so/self-hosting/editions-and-versions) to understand the difference between the Community and Commercial edition.

**Reply by pseudokool** [reply]
Feels ambiguous to me as well. The link above seems to say that self hosted comm. version has 12 seats.
However, when creating a workspace I have an option to provision for 500+ ... 
<img width="549" alt="image" src="https://github.com/user-attachments/assets/97b84747-2ada-4675-9339-e8ce5d4a4d47" />

Side note: Would love for the self hosted comm. version to be completely limitless - self hosting is generally painful, so most satisfied users would eventually move to paid. But, as a satisfied and so far successful self hoster, do I have an option to donate a variable amount as well, as there is a lot of effort and wouldn't mind paying back where possible.

**Reply by almereyda** [reply]
When searching the code for ["12"](https://github.com/search?q=repo%3Amakeplane%2Fplane+12&type=code) or ["500"](https://github.com/search?q=repo%3Amakeplane%2Fplane+500&type=code), only the latter results meaningful responses to this.

The number appears to be a meta datum, which is carried around. But I'm unable to find places where it is evaluated against a set of constraints.

https://github.com/makeplane/plane/blob/feb88e64a4a6ca239ac603ae1a6988d3f0c2b632/packages/constants/src/workspace.ts#L4-L11

Searching for ["organization size"](https://github.com/search?q=repo%3Amakeplane/plane%20ORGANIZATION_SIZE&type=code), we learn that there are multiple places in which it is used.

Previously it was called "company size":

https://github.com/makeplane/plane/blob/feb88e64a4a6ca239ac603ae1a6988d3f0c2b632/apiserver/plane/db/migrations/0035_auto_20230704_2225.py#L10

It is also a manually set variable, which does not carry any value about how many users are registered to an instance.

Searching in other places, I wouldn't find any codified policy which restricts sign ups on self-hosted instances. Maybe the limitation only exists in the licence, in order to be prepared in case one needed to be? I'd be interested if someone was able to find some kind of telemetry or other metadata about an instance, which leaves it without explicit consent.

For now it _seems_ we can use it. Did someone already try to register more than 12 accounts on their self-hosted instance?

**Reply by pseudokool** [reply]
This is helpful. Thanks. Was contemplating testing if the limits kick in.

However, after this post, came across another discussion here (can't find the link - will keep looking). Which talks about the limits: "... the apparent 12 limit on the community ver. .." is supposedly when performance starts getting affected "because of initial architecture decisions". So it looks like we are good to exceed this number bearing the disclaimer in mind.

**Reply by almereyda** [reply]
Yes, I had also noticed performance issues in the past, but since addition of the MQTT queue and more separation of concerns into separate microservices, think the live collaboration server, things have improved vastly on our self-hosted instance.

**Reply by theparthacus** [reply]
> @almereyda says,
>
> When searching the code for ["12"](https://github.com/search?q=repo%3Amakeplane%2Fplane+12&type=code) or ["500"](https://github.com/search?q=repo%3Amakeplane%2Fplane+500&type=code), only the latter results meaningful responses to this.
> 
> The number appears to be a meta datum, which is carried around. But I'm unable to find places where it is evaluated against a set of constraints.
> 
> https://github.com/makeplane/plane/blob/feb88e64a4a6ca239ac603ae1a6988d3f0c2b632/packages/constants/src/workspace.ts#L4-L11
> 
> Searching for ["organization size"](https://github.com/search?q=repo%3Amakeplane/plane%20ORGANIZATION_SIZE&type=code), we learn that there are multiple places in which it is used.
> 
> Previously it was called "company size":
> 
> https://github.com/makeplane/plane/blob/feb88e64a4a6ca239ac603ae1a6988d3f0c2b632/apiserver/plane/db/migrations/0035_auto_20230704_2225.py#L10
> 
> It is also a manually set variable, which does not carry any value about how many users are registered to an instance.

this number is a part of the metadata collected to understand the kinds of orgs that are signing up on Plane and how they plan to use it. this number has no bearing on plan restrictions whatsoever. that kicks in at the workspace level and only when you add 12 people to a workspace. try adding a 13th person and you should see an error. if you don't, please report the bug. 😉

>Searching in other places, I wouldn't find any codified policy which restricts sign ups on self-hosted instances. Maybe the limitation only exists in the licence, in order to be prepared in case one needed to be?

nope, nope. the restriction exists as a hard-coded limit with the Free plan on the Commercial Edition. the ethos of it is in line with what you say—to prevent shocks when upgrading.

> I'd be interested if someone was able to find some kind of telemetry or other metadata about an instance, which leaves it without explicit consent.

that doesn't happen outside the scope of our declared telemetry, but again, if you find an example, report it.

> @pseudokool says
>
> However, after this post, came across another discussion here (can't find the link - will keep looking). Which talks about the limits: "... the apparent 12 limit on the community ver. .." is supposedly when performance starts getting affected "because of initial architecture decisions"

there is no hard-coded restriction on the Community Edition, so the link or the conversation you are referring to must have talked about the Commercial Edition and why we mention an approximate restriction in some places. while all self-hosted editions come with some infra-related restrictions that can be pushed higher with an increase in infra, we have never talked about that limit for the Community Edition. because we didn't need to.

folks were worried there was an artificial restriction on the Commercial Edition, so we clarified there wasn't and any numbers we had anywhere was indicative of the threshold at which performance degradation was reported by our users. ↓ is taken from this live page as of today and clarifies our rationale very, very well.

![Performance restrictions by infra • Plane](https://github.com/user-attachments/assets/da715114-7236-4b90-a49f-8727b24b29de)

**Comment by UbiquitousBear** [comment]
If you're a bank, you're likely subject to regulations and SDLC scrutiny. You'll want to purchase Pro or Business (which require you to contact them). These are not free.

**Comment by mustafa0x** [comment]
Is the live collaboration server available in the free self hosted version of plane?

**Reply by theparthacus** [reply]
no, not yet.

**Reply by almereyda** [reply]
I must politely disagree. In our instance, we can collaborate on documents and updates are synced instantly to multiple clients. Also, the manifest now contains this:

https://github.com/makeplane/plane/blob/f70eae2f3be48b3cfb6ed579ef587c2a86a1c56b/deploy/selfhost/docker-compose.yml#L89-L100

**Reply by theparthacus** [reply]
sorry for the delay here. i am not around here as much as i was before. what you are talking about and showing is the collaboration on Pages only. i don't know why i assumed @mustafa0x was talking about collaboration throughout Plane, which is work items, cycles, modules, epics, and initiatives. that's not there right now.

Pages, though, do support real-time changes on Free and the Community Edition, and a little more on paid plans.

**Comment by TyronMYoung11** [comment]
Can someone answer?
has the self-hosted Community version, any seats/users count limit?

**Reply by theparthacus** [reply]
not at all

**Reply by TyronMYoung11** [reply]
@theparthacus Thanks

Please tell, Is the page https://plane.so/pricing refers to Commercial only?
Can you append Community version to the pricing page?

**Reply by theparthacus** [reply]
i thought i responded to that [above](https://github.com/orgs/makeplane/discussions/4532#discussioncomment-14472871), but clearly, it didn't post.

we will try to, Tyron. including a fifth column has been my life's biggest challenge, but i promise, i will wrangle this in my head if that's the least i do.

**Reply by TyronMYoung11** [reply]
@theparthacus

Thank you man! waiting for it!

**Reply by TyronMYoung11** [reply]
@theparthacus Man can you please update the page?
I want to start using Community Edition But i fear limits

**Reply by kingOf0** [reply]
@theparthacus Can we please get an answer to @TyronMYoung11's question?
>  Is the page https://plane.so/pricing refers to Commercial only?

**Reply by theparthacus** [reply]
> @theparthacus Man can you please update the page? I want to start using Community Edition But i fear limits

i can't update the page anytime soon, @TyronMYoung11, but i have already answered your question [here](https://github.com/orgs/makeplane/discussions/4532#discussioncomment-14478207). are there other answers that are blockers?

**Reply by theparthacus** [reply]
> @theparthacus Can we please get an answer to @TyronMYoung11's question?
> 
> > Is the page https://plane.so/pricing refers to Commercial only?

follow the thread, please, @kingOf0. the question is implicitly answered in our conversation about appending the Community Edition's features and limits on the Pricing page.

yes, [this page](https://plane.so/pricing?mode=self-hosted) talks about the Commercial Edition only.

**Reply by TyronMYoung11** [reply]
@theparthacus i wanna see the comparison table, check feature by feature

Also please care: The link you have send is not exists:
> yes, [this page](https://github.com/orgs/makeplane/discussions/plane.so/pricing?mode=self-hosted) talks about the Commercial Edition only.

Page not found - 404

**Reply by theparthacus** [reply]
fixed the link

the page was always the `Self-hosted` tab on our Pricing page.

**Comment by krowvin** [comment]
After reading through this thread I am still not sure if the self hosted version of Plane has a user limit. 

The website, if you click the "self hosted" tab, shows a user limit the same as the pro version:

<img width="595" height="762" alt="image" src="https://github.com/user-attachments/assets/1ddc25f1-13a7-4698-8525-72c8f9a16c5d" />


Via
https://plane.so/pricing

I assume the thread above where @theparthacus links to a page was the src of the page I linked above:
https://github.com/orgs/makeplane/discussions/4532#discussioncomment-15021869

Which is a 404 now probably due to a restructure since linking. 


Just in here to bump this as it has been ~4 months. This thread at least builds confidence for me to install Plane via Docker and try to smash a 1000 users in it and see how it goes!

**Reply by TyronMYoung11** [reply]
We had exactly the same question

**Reply by CrayonIRL** [reply]
They answer the question here:
https://developers.plane.so/self-hosting/editions-and-versions#why-we-separate-editions

“The Community Edition is completely open-source, with no restrictions beyond those outlined in the [AGPL v3.0 license](https://github.com/makeplane/plane/blob/preview/LICENSE.txt). This is the edition that is now ranking at #1 in our space on GitHub.”

**Reply by theparthacus** [reply]
> They answer the question here:
> https://developers.plane.so/self-hosting/editions-and-versions#why-we-separate-editions
> 
> “The Community Edition is completely open-source, with no restrictions beyond those outlined in the [AGPL v3.0 license](https://github.com/makeplane/plane/blob/preview/LICENSE.txt). This is the edition that is now ranking at #1 in our space on GitHub.”
> 
> 

yes. thank you, @CrayonIRL.

**Reply by theparthacus** [reply]
> After reading through this thread I am still not sure if the self hosted version of Plane has a user limit. 
> 
> The website, if you click the "self hosted" tab, shows a user limit the same as the pro version:
> 
> <img width="595" height="762" alt="image" src="https://github.com/user-attachments/assets/1ddc25f1-13a7-4698-8525-72c8f9a16c5d" />
> 
> 
> Via
> https://plane.so/pricing
> 
> I assume the thread above where @theparthacus links to a page was the src of the page I linked above:
> https://github.com/orgs/makeplane/discussions/4532#discussioncomment-15021869
> 
> Which is a 404 now probably due to a restructure since linking. 
> 
> 
> Just in here to bump this as it has been ~4 months. This thread at least builds confidence for me to install Plane via Docker and try to smash a 1000 users in it and see how it goes!

sorrry for that dud link

it wasn't any restructure. just something with GitHub and `https` instead of just `plane.so`.

fixed the link

it's the same as you took the screenshot from.
